pysoa/client/client.py

```diff
@@ -254,7 +254,7 @@ def call_action(
             **kwargs
         ).actions[0]
 
-    def call_actions(self, service_name, actions, expansions=None, raise_action_errors=True, **kwargs):
+    def call_actions(self, service_name, actions, expansion_query=None, raise_action_errors=True, **kwargs):
```

Comment:
I would probably leave this called `expansions` for consistency with our old stuff, but I agree with the other change to `expansion_config` so they're still named differently.

Reply:
I agree with this comment 100%. Otherwise this looks great.

Reply:
Agree. Updated.